site/gdocs/pages/AboutPage.tsx

```diff
@@ -8,12 +8,13 @@ import { ArticleBlocks } from "../components/ArticleBlocks.js"
 import Footnotes from "../components/Footnotes.js"
 
 export default function AboutPage({ content, slug }: OwidGdocAboutInterface) {
+    const shouldOverrideTitle = content["override-title"]
     return (
         <main className="about-page centered-article-container grid grid-cols-12-full-width">
             <h1 className="about-header col-start-2 col-end-limit display-2-semibold">
-                About
+                {(shouldOverrideTitle && content.title) || "About"}
             </h1>
-            <AboutNav slug={slug} />
+            <AboutNav slug={slug} hideNav={content["hide-nav"]} />
```

Comment:
I think it would be nicer, if we simply not render the `AboutNav` here, instead of passing in the arg. After all, that's the main way to decide whether something should be in the JSX or not.

```suggestion
            {!content["hide-nav"] && <AboutNav slug={slug} />
```